comments/stakeholder_annotations.md

```diff
@@ -23,6 +23,41 @@ This may then be further aggregated as desired -- for example, into an equivalen
 
 ## Process
 
+### Defining Categories
+
+Annotation categories are defined per map from the map data sheets.
+A new "Annotation Categories" sheet is added with the following structure:
```

Comment:
This is very useful as an example of how the process works and as a blueprint of what data/columns are required for the transform.  

However, if the intention is to have a single transform that handles annotations across a range of different maps, then I don't think that's going to work long-term because changing that single transform in the future could potentially break everything that uses it. Given the nature of our work where we tend to generate one-off datasets that are not often updated afterwards this is probably not a major concern, but I still think it's better to handle annotations on a per-map basis, similar to how views are currently done -- that way we can be confident that the transform side of things is fully atomic.

Reply:
Thanks! Yes, you're quite right on this one - come to think of it, single transform will not even work short-term, e.g. on maps that have different translations.